libd-llvm/src/d/llvm/expression.d

```diff
@@ -468,6 +468,7 @@ struct ExpressionGen {
 			auto thisPtr = LLVMBuildExtractValue(builder, dg, m.hasContext, "");
 			auto vtblPtr = LLVMBuildStructGEP(builder, thisPtr, 0, "");
 			auto vtbl = LLVMBuildLoad(builder, vtblPtr, "vtbl");
```

Comment:
However the real difference is only there if the function pointer obtained form the vtbl is marked invariant.